### PR TITLE
Windows install: Don't always reboot after vc redist

### PR DESCRIPTION
**Pull request by zanderz**
I had put in "scheduleReboot" before learning about testing for specific exit codes. The problem can still come up if the redist really does need a reboot, but this will cut down on it a lot.
@maxtaco @oconnor663

### CODE REVIEW

File: packaging/windows/WIXInstallers/KeybaseBundle/Bundle.wxs [hunk 1]
```diff
@@ -134,7 +136,9 @@
                   SourceFile="Redist\VCRedist_2015\vc_redist.x64.exe"
                   DownloadUrl="https://download.microsoft.com/download/6/3/a/63abaf83-2ca6-4460-90d7-12de8e815b1a/vc_redist.x64.exe"
                   >
-        <ExitCode Behavior="scheduleReboot"/>
+        <ExitCode Value="1641" Behavior="scheduleReboot"/>
+        <ExitCode Value="3010" Behavior="scheduleReboot"/>
+        <ExitCode Value="0" Behavior="success" />
```
Comment (oconnor663) [inline packaging/windows/WIXInstallers/KeybaseBundle/Bundle.wxs]: Should we add a comment or something to remind future us to remember to keep these two lists in sync?

File: packaging/windows/WIXInstallers/KeybaseBundle/Bundle.wxs [hunk 2]
```diff
@@ -114,7 +114,9 @@
                   SourceFile="Redist\VCRedist_2015\vc_redist.x86.exe"
                   DownloadUrl="https://download.microsoft.com/download/2/6/8/268f7be7-8d2a-42d6-a823-5c6941a3d70a/vc_redist.x86.exe"
       >
-        <ExitCode Behavior="scheduleReboot"/>
+        <ExitCode Value="1641" Behavior="scheduleReboot"/>
+        <ExitCode Value="3010" Behavior="scheduleReboot"/>
+        <ExitCode Value="0" Behavior="success" />
```
Comment (oconnor663) [inline packaging/windows/WIXInstallers/KeybaseBundle/Bundle.wxs]: Is there any worry that there might be some more obscure exit codes that also require rebooting? Maybe things we were handling before with our aggressive reboots, that could slip through now?

Comment (zanderz) [inline packaging/windows/WIXInstallers/KeybaseBundle/Bundle.wxs]: All other results are errors, which halt the installation. If anything, we may have been eating some of those before. There aren't any others we wouldn't want to cancel the install after: https://msdn.microsoft.com/en-us/library/windows/desktop/aa376931(v=vs.85).aspx

**Comment by oconnor663** [comment]
Cool beans. LGTM. (Maybe with that one comment request above.)

**Pull request merged**